modules/hivemq-edge-module-etherip/src/main/java/com/hivemq/edge/adapters/etherip/EipPollingProtocolAdapter.java

```diff
@@ -44,7 +49,7 @@
 
 public class EipPollingProtocolAdapter implements PollingProtocolAdapter<EipToMqttMapping> {
 
-    private static final @NotNull org.slf4j.Logger LOG = LoggerFactory.getLogger(EipPollingProtocolAdapter.class);
+    private static final @NotNull org.slf4j.Logger log = LoggerFactory.getLogger(EipPollingProtocolAdapter.class);
```

Comment:
org.slf4j.Logger => Logger

Reply:
we use log as the variable name everywhere in the code. And actually intellij even creates it as "log"